realkd/search.py

```diff
@@ -189,31 +191,62 @@ def from_df(df, without=None, max_col_attr=10, sort_attributes=True, discretizat
         >>> titanic_ctx.m
         891
         >>> titanic_ctx.attributes # doctest: +NORMALIZE_WHITESPACE
-        [Survived<=0, Survived>=1, Pclass<=1, Pclass<=2, Pclass>=2, Pclass>=3, Sex==male, Sex==female, Age<=23.0,
+        [Survived<=0, Survived>=1, Pclass<=1, Pclass<=2, Pclass>=2, Pclass>=3, Sex==female, Sex==male, Age<=23.0,
```

Comment:
REGRESSION:
Attributes created here will now be ordered alphabetically. I think this is a positive change

Reply:
Why is this a positive change? Why was it necessary/convenient?